company.py

```diff
@@ -0,0 +1,111 @@
+14. Вывести список всех сотредников с указанием зарплаты "на руки" и зарплаты с учётом налогов.
+15. Вывести список отделов, отсортированный по месячной налоговой нагрузки.
+16. Вывести всех сотрудников, за которых компания платит больше 100к налогов в год.
+17. Вывести имя и фамилию сотрудника, за которого компания платит меньше всего налогов.
+"""
+
+departments = [
+    {
+        "title": "HR department",
+        "employers": [
+            {"first_name": "Daniel", "last_name": "Berger", "position": "Junior HR", "salary_rub": 50000},
+            {"first_name": "Michelle", "last_name": "Frey", "position": "Middle HR", "salary_rub": 75000},
+            {"first_name": "Kevin", "last_name": "Jimenez", "position": "Middle HR", "salary_rub": 70000},
+            {"first_name": "Nicole", "last_name": "Riley", "position": "HRD", "salary_rub": 120000},
+        ]
+    },
+    {
+        "title": "IT department",
+        "employers": [
+            {"first_name": "Christina", "last_name": "Walker", "position": "Python dev", "salary_rub": 80000},
+            {"first_name": "Michelle", "last_name": "Gilbert", "position": "JS dev", "salary_rub": 85000},
+            {"first_name": "Caitlin", "last_name": "Bradley", "position": "Teamlead", "salary_rub": 950000},
+            {"first_name": "Brian", "last_name": "Hartman", "position": "CTO", "salary_rub": 130000},
+        ]
+    },
+]
+
+taxes = [
+    {"department": None, "name": "vat", "value_percents": 13},
+    {"department": "IT Department", "name": "hiring", "value_percents": 6},
+    {"department": "BizDev Department", "name": "sales", "value_percents": 20},
+]
+
+def money_position(departments):
+    dep_money = []
+    for employers in departments:
+        title = employers['title']
+        money = 0
+        for employers_info in employers['employers']:
+            money += employers_info['salary_rub']
+        dep_money.append(f'{title}: {money}')
+    return dep_money
+
+def position(departments):
+    job_title = []
+    for employers in departments:
+        for employers_info in employers['employers']:
+            if employers_info['salary_rub'] < 80000:
+                job_title.append(employers_info['position'])
+    return set(job_title)
+
+def dearest_employrs(departments):
+    name_dearest_employrs = []
+    for employers in departments:
+        for employers_info in employers['employers']:
+            if employers_info['salary_rub'] > 100000:
+                name_dearest_employrs.append(employers_info['first_name'])
+    return (name_dearest_employrs)
+
+def names_departmens_employ(departments):
+    name_dep_empl = []
+    for employers in departments:
+        depart = employers['title']
+        for employers_info in employers['employers']:
+            name = employers_info['first_name']
+            name_dep_empl.append(f'{depart} - {name}')
+    return (name_dep_empl)
+        
+def names_employees(departments):
+    names = []
+    for employers in departments:
```

Comment:
вводит в заблуждение
```suggestion
    for department in departments:
```